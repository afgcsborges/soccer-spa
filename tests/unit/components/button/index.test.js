import Button from 'components/button'
import React from 'react'
import { shallow } from 'enzyme'

it('should onClick event be called', () => {
    const onClickFn = jest.fn()

    const wrapper = shallow(<Button label="SOME_LABEL" onClick={onClickFn} />)

    expect(wrapper).toMatchSnapshot()
    wrapper.find('Styled(Button)').props().onClick()

    expect(onClickFn).toHaveBeenCalled()
})
