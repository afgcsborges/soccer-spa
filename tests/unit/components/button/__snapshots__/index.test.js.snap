// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should onClick event be called 1`] = `
<Styled(Button)
  onClick={[MockFunction]}
>
  SOME_LABEL
</Styled(Button)>
`;
